proto/desmos/subspaces/v1beta1/msgs.proto

```diff
@@ -0,0 +1,160 @@
+syntax = "proto3";
+package desmos.subspaces.v1beta1;
+
+import "gogoproto/gogo.proto";
+
+option go_package = "github.com/desmos-labs/desmos/x/staging/subspaces/types";
+
+// Msg defines subspaces Msg service.
+service Msg {
+
+  // CreateSubspace defines the method to create a subspace
+  rpc CreateSubspace(MsgCreateSubspace) returns (MsgCreateSubspaceResponse);
+
+  // EditSubspace defines the method to edit a subspace fields
+  rpc EditSubspace(MsgEditSubspace) returns (MsgEditSubspaceResponse);
+
+  // AddAdmin defines the method to add a new admin to the subspace
+  rpc AddAdmin(MsgAddAdmin) returns (MsgAddAdminResponse);
+
+  // RemoveAdmin defines the method to remove an admin from a specific subspace
+  rpc RemoveAdmin(MsgRemoveAdmin) returns (MsgRemoveAdminResponse);
+
+  // RegisterUser defines the method to let user posts inside a specific subspace
+  rpc RegisterUser(MsgRegisterUser) returns (MsgRegisterUserResponse);
+
+  // BlockUser defines the method to block user posts inside a specific subspace
+  rpc BlockUser(MsgBlockUser) returns (MsgBlockUserResponse);
```

Comment:
Since we're having `RegisterUser` and `BlockUser`, I think we should also have `UnregisterUser` and `UnblockUser` here. So that social networks can implement those functions as well.